test/functorch/test_aotdispatch.py

```diff
@@ -1020,9 +1020,7 @@ def f3(a):
         inp_clone = inp.clone()
         out_ref = f3(inp_ref_clone)
         out_test = f3_compiled(inp_clone)
-        # We rely on autograd's view-replay here, and view-replay gives up and uses as_strided
-        # for multi-output views
-        self.assertTrue(all('AsStridedBackward' in str(o.grad_fn) for o in out_test[:3]))
+        self.assertTrue(all('UnbindBackward' in str(o.grad_fn) for o in out_test[:3]))
```

Comment:
🤔 this might improve perf in some (rare) cases, if we're now avoiding `AsStridedBackward` for multi-output views in all cases.